### PR TITLE
Fix adapter doc grammatical errors

### PR DESCRIPTION
**Pull request by geetha-rangaswamaiah**
This PR fixes adapter doc grammatical errors.

### CODE REVIEW

File: site/_docs/file_adapter.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ sqlline> select * from sales.emps;
 ## Mapping tables
 
 Now for a more complex example. This time we connect to Wikipedia via
-HTTP, read pages for US states and cities, and extract data from HTML
+HTTP, read pages for US states and cities and extract data from HTML
```
Comment (julianhyde) [inline site/_docs/file_adapter.md:161]: The original (with comma) is correct.

See https://en.wikipedia.org/wiki/Eats,_Shoots_%26_Leaves.

File: site/_docs/file_adapter.md [hunk 2]
```diff
@@ -326,6 +326,6 @@ sqlline> select distinct deptno from depts;
 
 ## Future improvements
 
-We are continuing to enhance the adapter, and would welcome
+We are continuing to enhance the adapter and would welcome
```
Comment (julianhyde) [inline site/_docs/file_adapter.md:329]: Correct both with and without comma, but slightly clearer with it, given that there is an 'and' later in the sentence.

**Comment by rubenada** [comment]
Thanks for review this @geetha-rangaswamaiah . Could you please squash commits into a single one?

**Comment by geetha-rangaswamaiah** [comment]
> Thanks for review this @geetha-rangaswamaiah . Could you please squash commits into a single one?

Done.

**Comment by rubenada** [comment]
Thanks for the PR @geetha-rangaswamaiah !

**Pull request merged**